### ISSUE TITLE
MediaSync timeupdate delays

### ISSUE THREAD

**Issue by abdulzeedo** [webtiming/timingsrc#10]
Hi,

I was thinking that given the fact that I don't have sync perfection issues, I was trying to introduce some delays in timeupdate event handlers specified in mediasync.js so that the native seek UI command could work not only on generic players but Netflix and similars. 

What would be the cons and what would be the best place to introduce such delays?

**Comment by Snarkdoof** [comment]
I'm not sure what you are asking, but I don't believe there will be any
help in introducing delays anywhere. We're synchronizing Netflix and
similar perfectly with the current solution.

N
---
Dr. Njål Borch
Senior scientist
NORCE Tromsø, Norway

**Comment by abdulzeedo** [comment]
You are right. Upon some more testing, getting Netflix to synchronize has been pretty easy.

**Comment by Snarkdoof** [comment]
Good to hear :-)
